### ISSUE TITLE
Gap of unreleased helm chart versions that are in other repository

### ISSUE THREAD

**Issue by kang-makes** [newrelic/nri-kubernetes#473]
## Is your feature request related to a problem? Please describe.

During the migration of the helm charts there was a first iteration that make the charts hard to follow.

There is a helm-chart called `newrelic-infrastructure` chart that has versions 2 and 3 and a `newrelic-infrastructure-v3` that has some versions that are not even marked as beta.

`% helm search repo nr-helm-charts/newrelic-infrastructure --versions`
```
NAME                                              	CHART VERSION	APP VERSION	DESCRIPTION
nr-helm-charts/newrelic-infrastructure            	2.10.1       	2.11.0     	DEPRECATED: Moved to https://github.com/newreli...
nr-helm-charts/newrelic-infrastructure            	2.10.0       	2.11.0     	A Helm chart to deploy the New Relic Infrastruc...
nr-helm-charts/newrelic-infrastructure            	2.9.0        	2.10.0     	A Helm chart to deploy the New Relic Infrastruc...
nr-helm-charts/newrelic-infrastructure            	2.8.1        	2.10.0     	A Helm chart to deploy the New Relic Infrastruc...
nr-helm-charts/newrelic-infrastructure            	2.8.0        	2.9.0      	A Helm chart to deploy the New Relic Infrastruc...
[...]
nr-helm-charts/newrelic-infrastructure-v3         	3.1.0        	3.1.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.21       	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.18       	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.17       	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.16       	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.15       	3.0.2-pre  	A Helm chart to deploy the New Relic Kubernetes...
[...]
nr-helm-charts/newrelic-infrastructure-v3         	3.0.4        	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.3        	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.2        	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-helm-charts/newrelic-infrastructure-v3         	3.0.0        	3.0.0      	A Helm chart to deploy the New Relic Kubernetes...
```
`% helm search repo nr-nri-kubernetes/newrelic-infrastructure --versions`
```
NAME                                     	CHART VERSION	APP VERSION	DESCRIPTION
nr-nri-kubernetes/newrelic-infrastructure	3.5.1        	3.2.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-nri-kubernetes/newrelic-infrastructure	3.5.0        	3.1.1      	A Helm chart to deploy the New Relic Kubernetes...
nr-nri-kubernetes/newrelic-infrastructure	3.4.0        	3.1.1      	A Helm chart to deploy the New Relic Kubernetes...
[...]
nr-nri-kubernetes/newrelic-infrastructure	3.2.0        	3.2.0      	A Helm chart to deploy the New Relic Kubernetes...
nr-nri-kubernetes/newrelic-infrastructure	3.1.1        	3.1.1      	A Helm chart to deploy the New Relic Kubernetes...
nr-nri-kubernetes/newrelic-infrastructure	2.10.3       	2.13.0     	DEPRECATED: (In maintenance mode) Helm chart fo...
nr-nri-kubernetes/newrelic-infrastructure	2.10.2       	2.12.0     	DEPRECATED: (In maintenance mode) Helm chart fo...
```

After releasing those charts in this repository (at least to have a historic record) we have to manually remove the releases of `newrelic-infrastructure-v3` from the helm charts repository.

## Priority
Must Have

**Comment by paologallinaharbur** [comment]
What do you mean saying: "After releasing those charts in this repository (at least to have a historic record)"?
Do you mean to release all 3.x charts in `nri-kubernetes` repo? Would that be useful for users?

By the way I agree that we can and should get rid of "nr-helm-charts/newrelic-infrastructure-v3" and it would be nice to have them in the new repo. But I would not touch the releases and files, change change the index.yaml of the two repos (if possible)

**Comment by kang-makes** [comment]
Forks org is a playground. There is a chance that somebody removes all the releases there by mistake or testing something. Beta versions of `nri-bundle` could be lost.

If that is not an issue because they were beta releases then we can simply remove them from the index right now.

**Comment by paologallinaharbur** [comment]
I would ask @mangulonr to confirm, but in few months I would not see any problem getting rid of old beta releases if to keep them we need extra work.

Do you see any drawback caused by their removal? 

3.x charts of nri-kubernetes are a different things and we might want to migrate that

**Comment by kang-makes** [comment]
From my point of view, having it saved here allows doing a git blame and seeing changes in the files even knowing that there is a breakpoint in the history. 

But you are right, it is way less work to simply get rid of them.

Both `newrelic-infrastructure-v3` charts and `newrelic-infrastructure` with versions above 3.1.1 are beta and could be safely removed.
